interactive_ai/supported_models/geti_supported_models/manifests/classification/deit_tiny.yaml

```diff
@@ -0,0 +1,15 @@
+id: Classification_Deit_Tiny
```

Comment:
The ids must exactly match the ones in the model templates, for backward compatibility.
For example, this should be "Custom_Image_Classification_DeiT-Tiny"
https://github.com/open-edge-platform/training_extensions/blob/55f6153771987e18082cace1f6685e7380dabf91/src/otx/tools/templates/classification/deit_tiny/template.yaml#L2

Reply:
Ok, I thought we will start from scratch